docker/worker/worker_test.py

```diff
@@ -834,7 +834,7 @@ def setUp(self):
     self.maxDiff = None
     tests.reset_emulator()
 
-    self.original_clone = pygit2.clone_repository
+    self.original_clone = osv.clone
```

Comment:
osv.repos.clone ?

Reply:
osv.repos.clone is the internal import path. osv.clone is the external one.